src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -848,7 +867,7 @@ private void logConfiguration()
                value = dsProps;
             }
             value = (prop.contains("password") ? "<masked>" : value);
-            LOGGER.debug((prop + "................................................").substring(0, 32) + (value != null ? value : ""));
+            LOGGER.debug((prop + "................................................").substring(0, 32) + value);
```

Comment:
not allproperties, getNullIfEmpty() is applied only on catalog, connectionInitSql, connectionTestQuery, transactionIsolationName, dataSourceClassName, dataSourceJndiName, driverClassName and jdbcUrl